### PR TITLE
Remove list_session and change list_student to list

### LINKED ISSUE #134: Remove list_session command
list_session command is a repeated functionality of list_student command
Remove list_session command:
1. Remove from code base
2. Change 'list_student' to 'list'

### PR DESCRIPTION
**Pull request by enhao25**
Resolves #134

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/ListCommand.java [hunk 1]
```diff
@@ -8,11 +8,11 @@
 /**
  * Lists all students in the address book to the user.
```
Comment (samleewy) [inline src/main/java/seedu/address/logic/commands/ListCommand.java]: Not sure if should change to include sessions as well? so 'lists all students and sessions'

**Review by JonahhGohh — APPROVED** [review]
Yep LGTM!

**Review by samleewy — APPROVED** [review]
minor comment, else looks good!
another thing to note: has to be changed in DG as well, for whoever's doing it!

thanks!

File: src/main/java/seedu/address/logic/commands/ListCommand.java (hunk 1 above)
Comment (enhao25) [reply src/main/java/seedu/address/logic/commands/ListCommand.java]: Haha nice catch! Changed that in the latest commit liao!

**Pull request merged**